test/probot.test.ts

```diff
@@ -1,27 +1,69 @@
-import Stream from "stream";
-
 import {
   EmitterWebhookEvent,
   EmitterWebhookEvent as WebhookEvent,
 } from "@octokit/webhooks";
+
+import webhookExamples from "@octokit/webhooks-examples";
+import { EmitterWebhookEventName } from "@octokit/webhooks/dist-types/types";
 import Bottleneck from "bottleneck";
 import nock from "nock";
 import pino from "pino";
+import Stream from "stream";
+import { jest } from "@jest/globals";
 
-import { Probot, ProbotOctokit, Context } from "../src";
-
-import webhookExamples from "@octokit/webhooks-examples";
-import { EmitterWebhookEventName } from "@octokit/webhooks/dist-types/types";
+import { Context, Probot, ProbotOctokit } from "../src";
 
 const appId = 1;
 const privateKey = `-----BEGIN RSA PRIVATE KEY-----
-MIIBOQIBAAJBAIILhiN9IFpaE0pUXsesuuoaj6eeDiAqCiE49WB1tMB8ZMhC37kY
-Fl52NUYbUxb7JEf6pH5H9vqw1Wp69u78XeUCAwEAAQJAb88urnaXiXdmnIK71tuo
-/TyHBKt9I6Rhfzz0o9Gv7coL7a537FVDvV5UCARXHJMF41tKwj+zlt9EEUw7a1HY
-wQIhAL4F/VHWSPHeTgXYf4EaX2OlpSOk/n7lsFtL/6bWRzRVAiEArzJs2vopJitv
-A1yBjz3q2nX+zthk+GLXrJQkYOnIk1ECIHfeFV8TWm5gej1LxZquBTA5pINoqDVq
-NKZSuZEHqGEFAiB6EDrxkovq8SYGhIQsJeqkTMO8n94xhMRZlFmIQDokEQIgAq5U
-r1UQNnUExRh7ZT0kFbMfO9jKYZVlQdCL9Dn93vo=
+MIIJKAIBAAKCAgEAu0E+tR6wfOAJZ4lASzRUmvorCgbI5nQyvZl3WLu6ko2pcEnq
```

Comment:
can you clarify the goal of updating this key?

Reply:
> The key is copied from the other PR because a new format was required. https://github.com/probot/probot/pull/1821

ah. got it. thanks for the link to that other PR. looks like i need to get up to speed with those details as well